### PR TITLE
add test plan for dynamic audit

### PR DESCRIPTION
**Pull request by pbarker**
adds test plan for dynamic audit

### CODE REVIEW

File: keps/sig-auth/0014-dynamic-audit-configuration.md [hunk 1]
```diff
@@ -246,6 +246,10 @@ Benchmark testing will be required to understand the scope of the impact and wha
 is gated by opt-in feature flags, which allows it to move to alpha but these concerns must be tested and reconciled before it 
 progresses to beta.
 
+## Test Plan
+
+There are tests for the [plugin](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/plugin/pkg/audit/dynamic/dynamic_test.go) and any other changes to the codebase. There is an [integration test](https://github.com/kubernetes/kubernetes/blob/master/test/integration/master/audit_dynamic_test.go) as well as an [e2e test](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/auth/audit_dynamic.go).
+
```
Comment (tallclair) [inline keps/sig-auth/0014-dynamic-audit-configuration.md]: I think it's not required for beta, but I'd like to a see a scalability test before GA.

File: keps/sig-auth/0014-dynamic-audit-configuration.md [hunk 2]
```diff
@@ -246,6 +246,10 @@ Benchmark testing will be required to understand the scope of the impact and wha
 is gated by opt-in feature flags, which allows it to move to alpha but these concerns must be tested and reconciled before it 
 progresses to beta.
 
+## Test Plan
```
Comment (tallclair) [inline keps/sig-auth/0014-dynamic-audit-configuration.md]: nit: update the TOC

File: keps/sig-auth/0014-dynamic-audit-configuration.md [hunk 3]
```diff
@@ -246,6 +246,10 @@ Benchmark testing will be required to understand the scope of the impact and wha
...
+
+There are tests for the [plugin](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/plugin/pkg/audit/dynamic/dynamic_test.go) and any other changes to the codebase. There is an [integration test](https://github.com/kubernetes/kubernetes/blob/master/test/integration/master/audit_dynamic_test.go) as well as an [e2e test](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/auth/audit_dynamic.go).
```
Comment (tallclair) [inline keps/sig-auth/0014-dynamic-audit-configuration.md]: For beta, will the e2e test become a PR blocking test? The current audit e2e is disabled for being flaky, but I think the dynamic audit test should actually fix those problems.

Comment (pbarker) [reply keps/sig-auth/0014-dynamic-audit-configuration.md]: this is a good question, not sure who would be the one to answer that

**Pull request merged**